crates/nuts/Cargo.toml

```diff
@@ -9,12 +9,13 @@ serde_json = { workspace = true }
 thiserror = { workspace = true }
 url = { workspace = true }
 ciborium = { workspace = true }
-bitcoin = { workspace = true, features = ["base64", "rand-std"]}
+bitcoin = { workspace = true, features = ["base64", "rand-std"] }
 num-traits = { workspace = true }
 num-bigint = { workspace = true }
 once_cell = { workspace = true }
 tracing = { workspace = true }
 hex = { workspace = true }
+rusqlite = { workspace = true }
```

Comment:
make this optional, make it a feature and only do the ToSql FromSql behind the feature gate